core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -342,9 +347,7 @@ class ReplicaManagerTest {
       rm.becomeLeaderOrFollower(1, leaderAndIsrRequest2, (_, _) => ())
 
       assertTrue(appendResult.hasFired)
-    } finally {
-      rm.shutdown(checkpointHW = false)
-    }
+    } finally rm.shutdown(checkpointHW = false)
```

Comment:
should we be consistent with the new line vs no new line formatting?

Reply:
The test right now is a mix (even without my change). I have made it consistent now.